cli/README.md

```diff
@@ -72,6 +72,21 @@ To add user :
 run `node cli.mjs --addUser <userId>`
 example: node cli.mjs --addUser "123~A#netflix"
 
+## Downloading external git repository
+
+To download/clone external github repository and save repository contents in MF, below options can be used :
+
+`--downloadOverrides` - specify the url of a github repository to clone
```

Comment:
Inconsistent capitalization. This is lower case but '--overrideLocation' has "Specify" with upper case